devito/ops/types.py

```diff
@@ -17,6 +19,45 @@ def _C_typedata(self):
         return super()._C_typedata
 
 
+class RawAccessible(basic.Symbol):
+    is_Scalar = True
+
+    def __init_finalize__(self, name, read_only=True, **kwargs):
+        self.read_only = read_only
+        super().__init_finalize__(name, **kwargs)
+
+    @property
+    def _C_typename(self):
+        # TODO: can't change self.dtype from __init_finalize__
+        return '%s%s *' % (
+            'const ' if self.read_only else '',
+            ctypes_to_cstr(c_float)
+        )
+
+
+class RawAccess(basic.Basic, sympy.Basic):
+    def __init__(self, base, *args, **kwargs):
+        self.base = base
+        super().__init__(*args, **kwargs)
+
+    def __str__(self):
+        return '(*%s)' % self.base._C_name
+
+    def __repr__(self):
+        return self.base._C_name
+
+    def _eval_power(self, other):
+        # subclass to compute self**other for cases when
+        # other is not NaN, 0, or 1
+        return None
+
+    def as_coeff_Mul(self):
+        return sympy.S.One, self
+
+    def as_coeff_Add(self):
+        return sympy.S.Zero, self
```

Comment:
*I really want these ones gone.*
`Sympy`  operations should not be backend specific, it is what `Differentiable` and `basic` is for, this makes a lot of things not right and quite messy to fix.

Reply:
ok, removing! used that for a fix, but it seems it's not quite required indeed